.github/workflows/push_dockerhub.yml

```diff
@@ -0,0 +1,39 @@
+name: nf-core Docker push
+# This builds the docker image and pushes it to DockerHub
+# Runs on nf-core repo releases and push event to 'dev' branch (PR merges)
+on:
+  push:
+    branches:
+      - dev
+  release:
+    types: [published]
+
+push_dockerhub:
+  name: Push new Docker image to Docker Hub
+  runs-on: ubuntu-latest
+  # Only run for the nf-core repo, for releases and merged PRs
+  if: ${{ github.repository == 'nf-core/chipseq' }}
+  env:
+    DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+    DOCKERHUB_PASS: ${{ secrets.DOCKERHUB_PASS }}
+  steps:
+    - name: Check out pipeline code
+      uses: actions/checkout@v2
+
+    - name: Build new docker image
+      run: docker build --no-cache . -t nfcore/chipseq:latest
+
+    - name: Push Docker image to DockerHub (dev)
+      if: ${{ github.event_name == 'push' }}
+      run: |
+        echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+        docker tag nfcore/chipseq:latest nfcore/chipseq:dev
+        docker push nfcore/chipseq:dev
+
+    - name: Push Docker image to DockerHub (release)
+      if: ${{ github.event_name == 'release' }}
+      run: |
+        echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+        docker push nfcore/chipseq:latest
+        docker tag nfcore/chipseq:latest nfcore/chipseq:${{ github.event.release.tag_name }}
+        docker push nfcore/chipseq:${{ github.event.release.tag_name }}
```

Comment:
Full change with whitespace:
```suggestion
jobs:
  push_dockerhub:
    name: Push new Docker image to Docker Hub
    runs-on: ubuntu-latest
    # Only run for the nf-core repo, for releases and merged PRs
    if: {% raw %}${{{% endraw %} github.repository == '{{ cookiecutter.name }}' {% raw %}}}{% endraw %}
    env:
      DOCKERHUB_USERNAME: {% raw %}${{ secrets.DOCKERHUB_USERNAME }}{% endraw %}
      DOCKERHUB_PASS: {% raw %}${{ secrets.DOCKERHUB_PASS }}{% endraw %}
    steps:
      - name: Check out pipeline code
        uses: actions/checkout@v2

      - name: Build new docker image
        run: docker build --no-cache . -t {{ cookiecutter.name_docker }}:latest

      - name: Push Docker image to DockerHub (dev)
        if: {% raw %}${{ github.event_name == 'push' }}{% endraw %}
        run: |
          echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
          docker tag {{ cookiecutter.name_docker }}:latest {{ cookiecutter.name_docker }}:dev
          docker push {{ cookiecutter.name_docker }}:dev

      - name: Push Docker image to DockerHub (release)
        if: {% raw %}${{ github.event_name == 'release' }}{% endraw %}
        run: |
          echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
          docker push {{ cookiecutter.name_docker }}:latest
          docker tag {{ cookiecutter.name_docker }}:latest {{ cookiecutter.name_docker }}:{% raw %}${{ github.event.release.tag_name }}{% endraw %}
          docker push {{ cookiecutter.name_docker }}:{% raw %}${{ github.event.release.tag_name }}{% endraw %}
```

Reply:
Sorry. Just saw this but I think I got I have sorted it.